### PR TITLE
Add option to run session without mir on mir (wayland session)

### PR DESCRIPTION
**Pull request by mariogrip**
This splits mir and wayland into two own sessions.

As https://gitlab.com/ubports/core/seeds/ubuntu-touch/merge_requests/7 got merged, this will be needed too.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -15,13 +15,29 @@ Depends: libpam-runtime (>= 1.0.1-6),
...
+Description: Mir on Mir session using unity-system-compositor
+ Mir on Mir session using unity-system-compositor
+
+Package: ubuntu-touch-session-wayland
+Architecture: all
+Depends: ubuntu-touch-session (= ${binary:Version}),
+         qtwayland5
+Conflicts: ubuntu-touch-session-mir
+Description: Wayland session without using mir on mir
+ Wayland session without using mir on mir
```
Comment (UniversalSuperBox) [inline debian/control]: Needs a trailing newline

**Review by UniversalSuperBox — CHANGES_REQUESTED** [review]
I assume this needs to be released alongside an update to the metapackage or else everything explodes

File: debian/control (hunk 1 above)
Comment (mariogrip) [reply debian/control]: fixed

**Comment by mariogrip** [comment]
> I assume this needs to be released alongside an update to the metapackage or else everything explodes

Well it shouldn't as mir is listed as the first dependency, but it will explode the other way around (if the meta got merged before this). but it's good to merge them at the same time anyway

**Review by UniversalSuperBox — APPROVED** [review]
Upgrading to this package version correctly installed `ubuntu-touch-session-mir` as well.

**Pull request merged**